microsoft-edge/devtools-guide-chromium/javascript/breakpoints.md

```diff
@@ -134,28 +140,25 @@ To set a DOM change breakpoint:
 
 
 <!-- ====================================================================== -->
-## XHR/Fetch breakpoints
+## XHR/fetch breakpoints
 
-Use an XHR breakpoint when you want to break when the request URL of an XHR contains a specified string.  DevTools pauses on the line of code where the XHR runs the `send()` method.
+Use an XHR/fetch breakpoint when you want to break when an [XmlHttpRequest](https://developer.mozilla.org/docs/Web/API/XMLHttpRequest) (XHR) or [Fetch](https://developer.mozilla.org/docs/Web/API/Fetch_API) request occurs. DevTools pauses on the line of code where the XHR or Fetch request occurs.
 
-> [!NOTE]
-> This feature also works with [Fetch API](https://developer.mozilla.org/docs/Web/API/Fetch_API) requests.
-
-One example of when this is helpful is when your webpage is requesting an incorrect URL, and you want to quickly find the AJAX or Fetch source code that is causing the incorrect request.
+One example of when this is helpful is when your webpage is requesting an incorrect URL, and you want to quickly find the XHR or Fetch source code that is causing the incorrect request.
 
 To set an XHR breakpoint:
 
 1. Select the **Sources** tool.
 
-1. Expand the **XHR Breakpoints** panel.
+1. Expand the **XHR/fetch Breakpoints** panel.
 
 1. Click **Add breakpoint**.
 
-1. Enter the string which you want to break on.  DevTools pauses when this string is present anywhere in an XHR request URL.
+1. Enter all or part of the URL which you want to break on.  DevTools pauses when the value you enter is present anywhere in an XHR or Fetch request URL.
```

Comment:
```suggestion
1. Enter all or part of the URL that you want to break on.  DevTools pauses when the value you enter is present anywhere in an XHR or Fetch request URL.
```
according to style guide
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/that-vs-which

Reply:
done